### PR TITLE
KEYCLOAK-9670: Skeleton of the newsletter

### PR DESCRIPTION
**Pull request by sebastienblanc**
@stianst Can you check if it looks okay ?

### CODE REVIEW

File: blog/keycloak-newsletter-1.ftl [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<#--
+title = Keycloak Community Newsletter #1
+date = 2019-03-11
+publish = true
```
Comment (stianst) [inline blog/keycloak-newsletter-1.ftl]: Changed to false. Otherwise the empty newsletter would be posted ;)

**Comment by stianst** [comment]
Overall looks OK, but hard to say until there's some actual stuff here ;)

One thing you need to change publish to false so it's not published until it's ready to go.

**Comment by sebastienblanc** [comment]
@stianst With the publish flag set on false it stills appears on my local generated website, is that something that handled on the hosted side ?

**Comment by stianst** [comment]
Just fixed it here:
https://github.com/keycloak/keycloak-web/pull/47

If you run build with -Dpublish it will not include draft blogs. Also, that will make it use full URLs for things rather than file urls ;)

**Comment by sebastienblanc** [comment]
@stianst Nice, so maybe we can merge this and then people (and me) can just submit PRs to add content to the newsletter, wdyt ?

**Pull request merged**